apps/src/studio/api.js

```diff
@@ -166,6 +166,14 @@ exports.changeScore = function (id, value) {
   Studio.queueCmd(id, 'changeScore', {'value': value});
 };
 
+exports.getScore = function () {
+  return Studio.playerScore;
+};
+
+exports.setScore = function (value) {
+  Studio.setScore(value);
```

Comment:
@joshlory: Are you reasonably familiar with the command queue? Does this look right to you?